Resources/Prototypes/_White/Entities/Effects/puddle.yml

```
@@ -0,0 +1,37 @@
+          mask:
+            - ItemMask
+          layer:
+            - SlipLayer
+          hard: false
+    - type: SolutionContainerManager
+      solutions:
+        step: { maxVol: 2 }
+    - type: FootPrint
+    - type: Puddle
+      solution: step
+    - type: Appearance
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)